_about/release-cadence.md

```diff
@@ -33,24 +34,25 @@ we will increase the product's version number only for LTS releases.
 Our naming scheme for LTS releases is:
 
 ```plain
-<major>.<minor>.<patch level>
+<major>.<minor>.<LTS patch level>
 ```
 
-where `minor` is increased for every LTS release, and `patch level` counts the number of patches for the
+where `minor` is increased for every LTS release, and `LTS patch level` counts the number of patches for the
 current LTS release.
 
 For Snapshot releases, our numbering scheme is:
 
 ```plain
-<major>.<minor>.<patch level>-snapshot.<snaoshot count>
+<major>.<minor>.<LTS patch level>-snapshot.<snapshot count>
 ```
 
-where `major` and `minor` represent the number of the next LTS release, `patch level` counts the number of
-patches for the Snapshot release, and `snapshot count` starts at 0 and increases for every Snapshot release leading up
-to the next LTS.
+where `major` and `minor` and `LTS patch level` represent the next LTS, and
```

Comment:
that's clearer thank you, but you might as well replace <LTS patch level> by 0 as before an LTS release happens the patch level will always be 0
(though we may do -rc for patch levels I suppose but those wouldn't be monthlies anyway)

I would recommend to replace "snapshot" with "rc" in the string for both familiarity and brievety